spiffworkflow-backend/src/spiffworkflow_backend/__init__.py

```diff
@@ -25,13 +25,12 @@
 from spiffworkflow_backend.services.monitoring_service import configure_sentry
 from spiffworkflow_backend.services.monitoring_service import setup_prometheus_metrics
 
-# This is necessary if you want to use use the pymysql library with sqlalchemy rather than mysqlclient.
-# This is only potentially needed if you want to run non-docker local dev.
-# See the repo's top-level README and the linked troubleshooting guide for details.
+# This is necessary to make sure that the pymysql library is used as the MySQLdb library
+# This is only needed if you want to run non-docker local and are using Python 3. 
+# See the repo's top-level README for details.
```

Comment:
Is this a mac related issue? Before a week ago or so i ran this all local with mysql on linux and never had these issues.

Reply:
Also just to note we don't support anything below python 3.10 (iirc), but definitely not a python below 3.